src/components/PopUp/PopUpDonation.js

```diff
@@ -55,9 +56,12 @@ class PopUpDonation extends PopUp {
     const sectorsItems = (this.model.get('sectors').length > 0) ? this._getSectors() : '';
     const regionsItems = (this.model.get('countries').length > 0) ? this._getRegions() : '';
 
-    return `<div class="wrapper -donations"><header class="donation-header"><h1 class="text text-module-title -light"> ${ utils.numberNotation( (this.model.get('total_donors')) ) } donations -
-      <span class="number-m">${ utils.numberNotation(  this.model.get('total_funds') )}$</span></h1>
-      <div id="btn-donors" class="btn-donors js-popup-donation"><p>+</p></div></header>
+    return `<div class="wrapper -donations">
+      <header class="donation-header">
+        <h1 class="text text-module-title -light"> ${ this.model.attributes.donors.length === 1 ? this.model.attributes.donors[0]['name'] : utils.numberNotation( (this.model.get('total_donors')) ) + ' donations ' } -
+        <span class="number-m">$${ utils.numberNotation(  this.model.get('total_funds') )}</span></h1>
+        ${ this.model.attributes.donors.length > 1 ? '<div id="btn-donors" class="btn-donors js-popup-donation"><p>+</p></div>' : '' }
+      </header>
```

Comment:
Can you take this logic out of return function as we do for ${ sectorsItems } ? 
I think this way is easier to follow.